packages/client/tests/functional/decimal/tests.ts

```diff
@@ -83,7 +83,8 @@ testMatrix.setupTestSuite(
       })
 
       // https://github.com/prisma/prisma/issues/5925
-      test.failing('preserves precision when writing shorter numbers to to db', async () => {
+      // fails on sqlite, because sqlite decimal is actually a float
```

Comment:
Does this point to a bigger problem for SQLite? Is just accepting this (and not running the test) ok? Should we somehow deal with Decimals and SQLite differently?

Reply:
I don't think we should do anything differently for SQLIte, this is mostly on SQLIte itself and not on us. Basically, their `NUMERIC` type [always ends up being either int or float](https://www.sqlite.org/datatype3.html#:~:text=A%20column%20with%20NUMERIC%20affinity%20may%20contain%20values%20using%20all%20five%20storage%20classes.%20When%20text%20data%20is%20inserted%20into%20a%20NUMERIC%20column%2C%20the%20storage%20class%20of%20the%20text%20is%20converted%20to%20INTEGER%20or%20REAL%20(in%20order%20of%20preference)) and there is no real arbitrary precision decimal types (`DECIMAL` is just an alias for `NUMERIC`), just [an optional extension](https://sqlite.org/floatingpoint.html#the_decimal_c_extension) for working with  strings as if they were decimals. So, imho, it is very similar situations to enums - maybe we could tell `migrate` to store them as strings, but then we won't be able to re-introspect them.

With that in mind, I think it's ok to just skip the test, as we are not dealing with decimals any worth then underlying DB does.